src/main/java/brandkon/ProductService.java

```diff
@@ -0,0 +1,65 @@
+package brandkon;
+
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class ProductService {
+
+    private ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    //상품 목록 조회
+    public List<ProductResponse> getAllProducts(){
+        List<ProductResponse> responses = productRepository.findAll()
+                .stream()
+                .map(product ->
+                        new ProductResponse(product.getId(), product.getBrand().getName() ,product.getName(), product.getPrice(),""))
+                .toList();
+        return responses;
+    }
+
+    public List<ProductResponse> getAllProductsByBrandId(long brandId){
+        List<ProductResponse> responses = productRepository.findByBrandId(brandId)
+                .stream()
+                .map(product ->
+                        new ProductResponse(product.getId(), product.getBrand().getName() ,product.getName(), product.getPrice(),""))
+                .toList();
+        return responses;
+    }
+
+    //인기 상품 목록 조회
+    public List<ProductResponse> getPopularProducts(long categoryId, long brandId){
+        List<Product> products;
+        if(!(categoryId==0)) {
```

Comment:
사소한 것이지만 아래 방식이 더 일반적인 것 같아요
```suggestion
        if(categoryId!=0) {
```